fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -4761,7 +4761,7 @@ public void readFromJson(String json) throws IOException {
                         field.set(this, root.get(attr.name()));
                         break;
                     case "double":
-                        field.set(this, root.get(attr.name()));
+                        field.set(this, Double.valueOf(root.get(attr.name()).toString()));
```

Comment:
why change this?

Reply:
root.get(attr.name()) This is a double string. If it is not forced to be converted, an error will be reported. The int type has been forced to be converted before.